packages/forms-web-app/src/controllers/examination/file-upload/fileValidation.js

```diff
@@ -0,0 +1,57 @@
+const { calcMaxFileSizeLimit } = require('./utils');
+
+const allowedMimeTypes = [
```

Comment:
There are a few extra ones I think, some may have been added later eg DOCX.  James has these in his PR:
		fileSizeLimit: 52428801,
		allowedFileTypes: {
			pdf: 'application/pdf',
			doc: 'application/msword',
			docx: 'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
			jpg: 'image/jpeg',
			jpeg: 'image/jpeg',
			png: 'image/png',
			tif: 'image/tiff',
			tiff: 'image/tiff',
			xls: 'application/vnd.ms-excel',
			xlsx: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'

Reply:
Updated to: 

const allowedMimeTypes = [
	'application/pdf',
	'application/msword',
	'application/vnd.ms-excel',
	'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
	'image/jpeg',
	'image/jpeg',
	'image/png',
	'image/tiff',
	'image/tiff',
	'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet'
];

Reply:
File input button will only
 accept - attributes: { multiple: 'multiple', accept: ".pdf,.doc,.docx,.jpg,.jpeg,.png,.tif,.tiff,.xls,.xlsx", type: "file" },

Reply:
the no JS will catch all regardless if the html misses it.